pages/dkp/kommander/1.4/projects/index.md

```diff
@@ -9,40 +9,44 @@ excerpt: Multi-cluster Configuration Management
 
 # Projects overview
 
-Federation, in this context, means that a common configuration is pushed out from a central location (Kommander) to all Kubernetes clusters, or a pre-defined subset group under Kommander management. That pre-defined subset group of Kubernetes clusters is called a Project.
+Federation, in this context, means that a central location (Kommander) hosts and pushes common configurations to all Kubernetes clusters, or a pre-defined subset group under Kommander management. That pre-defined subset group of Kubernetes clusters is called a **Project**.
```

Comment:
I don't need a project to federate stuff using the default workspace.

Reply:
I'm not sure I understand your suggestion here. Would you mind providing a more accurate wording, please?

Reply:
Plus, this PR doesn't specifically change the wording around the issue you are (supposedly) addressing. Maybe a subsequent PR should be issued?

Reply:
I was sharing a detail, I think this statement was omitting. I am not sure if the intention of this Pr is to address all these possible details.

Reply:
No, as the title suggests this PR is mainly about adding details on kubefed placement selectors to the documentation.

Reply:
I'd be happy to see this page further improved, though, @hectorj2f.

Reply:
Sure, continue improving it 👍🏻